scrapbook/utils.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+utils.py
+
+Provides the base API calls for scrapbook
+"""
+import sys
+import warnings
+
+from functools import wraps
+
+
+def is_kernel():
+    '''
+    Returns True if execution context is inside a kernel
+    '''
+    in_ipython = False
```

Comment:
what's the distinction between these two variables?

Reply:
the first checks that you're in python, probably doesn't need to be assigned to a variable.